src/store/__mocks__/load-data.js

```diff
@@ -1,7 +1,8 @@
 import { getUrl } from '../../utils';
 import animals from '../../utils/data/animals.mock.json';
 import demo from '../../utils/data/demo.mock.json';
-import node_task from '../../utils/data/node_task.mock.json';
+import nodeTask from '../../utils/data/node_task.mock.json';
```

Comment:
see above comment about the casing - the naming should maintain as a snake case to follow the naming of the file

Reply:
@bru5  asked me to change to the camel case. I can't reference his comment anymore :S ... can't find it.

Reply:
aargh I see - yea tbh this change request is out of scope for this PR, and now I see where this is coming from - this is worth a discussion, but would be out of scope for this PR and we can save it for another time.